packages/geolocator/tizen/src/app_settings_manager.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_APP_SETTINGS_MANAGER_H_
+#define FLUTTER_PLUGIN_APP_SETTINGS_MANAGER_H_
+
+#include <app_control.h>
+
+#include <string>
+
+class AppSettingsManager {
+ public:
+  AppSettingsManager() {}
+  ~AppSettingsManager() {}
+
+  bool OpenAppSettings();
+  bool OpenLocationSettings();
+};
```

Comment:
If this class does not have any member variables and does not have any status, why should this class be provided as a class which It needs to be instantiated?